be1-go/db/sqlite/cli/schema.sql

```diff
@@ -51,4 +51,60 @@ CREATE TABLE message_witness (
     message_id TEXT NOT NULL
         REFERENCES message_info(message_id)
         ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+--
+-- Election channel
+--
+
+CREATE TABLE election_channel (
+    election_channel_id TEXT NOT NULL PRIMARY KEY,
+
+    start_timestamp INTEGER,
+    end_timestamp INTEGER,
+    terminated INTEGER
+);
+
+CREATE TABLE election_attendee (
```

Comment:
The voters are the attendees of the roll call.

Reply:
We could add a foreign key to a `lao_channel`. Then from that we have access to the `lao_channel` attendees. However an election channel is only provided it's channel ID (which is more a channel path, but...), which is composed of the laoID and the election ID. So that means the election channel should parse its channel ID and extract the lao ID from it. So at the end, and election should be provided its ID + the lao ID, and then the election channel can build it channel path.

I see your point of having a clean schema. However if we do that it require to reflect those - totally full of sense and beneficial - changes in the implementation. Again now the storage implementation "dummily" maps the implementation. So as such there is no schema to maintain, since it only require to map the implementation. Which is why the schema is not "clean" on its own, because it maps the implementation. I would be happy to make those change, if time permits, in another PR. But the scope of this PR is to implements a mapping between an election channel and a relational DB.